iot-hub/Samples/service/CleanupDevicesSample/CleanupDevicesSample.cs

```diff
@@ -37,7 +41,7 @@ public CleanupDevicesSample(RegistryManager rm, BlobContainerClient sc, List<str
         {
             _registryManager = rm ?? throw new ArgumentNullException(nameof(rm));
             _blobContainerClient = sc ?? throw new ArgumentNullException(nameof(sc));
-
+            Console.WriteLine($"Delete devices with prefixes: {JsonConvert.SerializeObject(deleteDevicesWithPrefix)}");
```

Comment:
Why call `JsonConvert.SerializeObject()` on a list of strings?

Reply:
I just want to print all the values in the list. I could use string.Join or linq but just chose one way to print. If there is an argument for why one way is better that the other, I can change it.

Reply:
Ah, I see. Yeah, I was a bit confused but now I get it. I think string.join would be more straight forward, but I think this is acceptable as well.